src/main/java/uk/gov/digital/ho/hocs/info/api/CountryService.java

```diff
@@ -23,24 +25,28 @@ public CountryService(CountryRepository countryRepository, ListConsumerService l
         this.listConsumerService = listConsumerService;
     }
 
-    @Cacheable(value = "ActiveCountrys", unless = "#result == null or #result.size() == 0")
+    @Cacheable(value = "ActiveCountrys", unless = "#result == null or #result.size() == 0" )
```

Comment:
Are these five white space changes done by your IDE?

Reply:
yes, I use these commands frequently:
option + command + i
option + command + o
option + command + l

to reformat the code etc.

Reply:
Yeah, but it looks weird!